packages/contentstack-import/src/import/modules/extensions.ts

```diff
@@ -109,19 +125,88 @@ export default class ImportExtensions extends BaseClass {
     );
   }
 
-  /**
-   * @method serializeExtensions
-   * @param {ApiOptions} apiOptions ApiOptions
-   * @returns {ApiOptions} ApiOptions
-   */
-  serializeExtensions(apiOptions: ApiOptions): ApiOptions {
-    const { apiData: extension } = apiOptions;
-    if (this.extUidMapper.hasOwnProperty(extension.uid)) {
-      log(this.importConfig, `Extension '${extension.title}' already exists. Skipping it to avoid duplicates!`, 'info');
-      apiOptions.entity = undefined;
-    } else {
-      apiOptions.apiData = extension;
-    }
-    return apiOptions;
+  async replaceExtensions(): Promise<any> {
+    const onSuccess = ({ response, apiData: { uid, title } = { uid: null, title: '' } }: any) => {
+      this.extSuccess.push(response);
+      this.extUidMapper[uid] = response.uid;
+      log(this.importConfig, `Extension '${title}' replaced successfully`, 'success');
+      fsUtil.writeFile(this.extUidMapperPath, this.extUidMapper);
+    };
+
+    const onReject = ({ error, apiData }: any) => {
+      this.extFailed.push(apiData);
+      log(this.importConfig, `Extension '${apiData.title}' failed to replace ${formatError(error)}`, 'error');
+      log(this.importConfig, error, 'error');
+    };
+
+    await this.makeConcurrentCall(
+      {
+        apiContent: this.existingExtensions,
+        processName: 'Replace extensions',
+        apiParams: {
+          reject: onReject.bind(this),
+          resolve: onSuccess.bind(this),
```

Comment:
[MINOR] Need not bind the current object into it. However, the current code will still work the way we expect.

Reply:
@antonyagustine , will take this on phase2, since this is done for all modules, we need to update everywhere and test again